hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -114,8 +113,8 @@
   /** Priority buckets */
   @VisibleForTesting
   static final float DEFAULT_SINGLE_FACTOR = 0.25f;
-  static final float DEFAULT_MULTI_FACTOR = 0.50f;
-  static final float DEFAULT_MEMORY_FACTOR = 0.25f;
+  private static final float DEFAULT_MULTI_FACTOR = 0.50f;
```

Comment:
For purely cosmetic/checkstyle fixes, could we do it on a separate jira?

Reply:
@wchevreuil since the checkstyle fixes are very less, is it fine to send them with this PR? I would take care of this going forward

Reply:
Updated this PR with only unit tests. Will create separate JIRA for cosmetic/checkstyle fixes

Reply:
Yeah, having specific jiras for each fix is generally preferred.